## 03_text_files.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
## Description: Remove encoding tag
--- a/src/knmap.desktop
+++ b/src/knmap.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=knmap
 Name[xx]=xxknmapxx
 Exec=knmap
